tests/Go/PhpUnit/ClassLocator.php

```diff
@@ -0,0 +1,147 @@
+<?php
+/*
+ * Go! AOP framework
+ *
+ * @copyright Copyright 2011, Lisachenko Alexander <lisachenko.it@gmail.com>
+ *
+ * This source file is subject to the license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Go\PhpUnit;
+
+use Go\Core\AspectContainer;
+use Go\Instrument\PathResolver;
+use Go\ParserReflection\LocatorInterface;
+use Go\ParserReflection\ReflectionEngine;
+use ReflectionClass;
+use ReflectionProperty;
+
+final class ClassLocator implements LocatorInterface
```

Comment:
[NEED HELP!] You can check this class, but in general, I have did my debugging, it works, locates files as I intended to be located.